_posts/2021-04-18-api-conversion.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "API versions conversion"
+date: 2021-04-18
+author: jakub_stejskal
+---
+Upgrade process of Strimzi and Kafka is not trivial.
```

Comment:
I would not refer to this as the upgrade process ... I would probably remove this sentence.

Reply:
Ye, it's leftover